apps/payments/next/lib/utils/pageMetadata.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import { headers } from 'next/headers';
+ import { fetchCMSData } from '@fxa/payments/ui/actions';
+ import { buildRedirectUrl } from '@fxa/payments/ui';
+ import { config } from 'apps/payments/next/config';
+ import type { Metadata } from 'next';
+ import type { CheckoutParams, } from '@fxa/payments/ui/server';
+
+  type Page =
+    | 'landing'
+    | 'new'
+    | 'start'
+    | 'success'
+    | 'error'
+    | 'location'
+    | 'page-not-found'
+    | 'processing'
+    | 'needs_input';
+  type PageType = 'checkout' | 'upgrade';
```

Comment:
Move these into `libs/payments/ui/src/lib/utils/types.ts`, and replace the types used in `libs/payments/ui/src/lib/utils/buildRedirectUrl.ts` with references to your types